tests/interchaintest/setup.go

```diff
@@ -25,7 +25,7 @@ var (
 		Bech32Prefix:        "juno",
 		Denom:               "ujuno",
 		CoinType:            "118",
-		GasPrices:           "0.003ujuno",
+		GasPrices:           "0.0ujuno",
```

Comment:
ujunox?

Reply:
for ictest we use standard ujuno, not x

Reply:
per `Denom:               "ujuno",` for the base denom